AppController/lib/helperfunctions.rb

```diff
@@ -875,6 +875,14 @@ def self.generate_secure_location_config(handler, port)
     return result
   end
 
+  def self.get_loaded_apps()
```

Comment:
In my mind not, since we use it to remove the routing config. We cross-check with the UserAppServer in the djinn. So the file system is ground truth for the config.